src/components/Input/Input.jsx

```diff
@@ -33,17 +33,20 @@ Input.defaultProps = {
   onChange: () => {},
   onFocus: () => {},
   error: '',
+  placeholder: '',
 };
 
+
 Input.propTypes = {
   type: PropTypes.string.isRequired,
   value: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired,
-  placeholder: PropTypes.string.isRequired,
+  placeholder: PropTypes.string,
```

Comment:
@ayodejiAA Please, why did you remove the placeholder attribute as a required prop? I feel placeholder should be a  required prop because since our form inputs don't have labels, we need a way to give users a clue of the type of information they are required to fill in the input.